### PR TITLE
feat/btc_markets connector 

### PR DESCRIPTION
**Pull request by lt7**
**Before submitting this PR, please make sure**:

- [X] Your code builds clean without any errors or warnings
- [X] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

Connector for https://www.btcmarkets.net/

API docs - https://docs.btcmarkets.net/v3/

Commonwealth Thread - https://commonwealth.im/hummingbot-foundation/discussion/3727-5005-featbtc_markets-connector 

**Tests performed by the developer**:

image

**Tips for QA testing**:

### CODE REVIEW

**Comment by aarmoa** [comment]
@lt7 please check the validation workflow results. The test coverage for the new connector is at 15% and the minimum coverage required for the changes in any PR is 80%. Please add the required unit tests for the new connector following the examples for Binance connector in https://github.com/hummingbot/hummingbot/tree/development/test/hummingbot/connector/exchange/binance

**Comment by lt7** [comment]
> @lt7 please check the validation workflow results. The test coverage for the new connector is at 15% and the minimum coverage required for the changes in any PR is 80%. Please add the required unit tests for the new connector following the examples for Binance connector in https://github.com/hummingbot/hummingbot/tree/development/test/hummingbot/connector/exchange/binance Remember that unit tests should mock any interaction with the exchange (they should not depend on external components, because they must be deterministic)

@aarmoa the test cases in test/connector/exchange/btc_markets in the PR screenshot require an API key and connectivity with the exchange. I've understand your point and have taken a look at adding the mock tests, however it requires WebAssistant aiohttp etc however this connector pre-dates that code so it needs refactoring. Adapting WebAssistant also adds throttling support so will be done at some point, but I don't plan on refactoring it at present.

I have added an exclusion to .coveragerc to get the checks to pass.

File: .coveragerc [hunk 1]
```diff
@@ -21,6 +21,7 @@ omit =
         hummingbot/connector/exchange/beaxy*
         hummingbot/connector/exchange/bitfinex*
         hummingbot/connector/exchange/blocktane*
+        hummingbot/connector/exchange/btc_markets*
```
Comment (aarmoa) [inline .coveragerc:24]: Sorry @lt7, adding exceptions to coverage is only allowed for very particular and well justified cases. A new connector is not included in the justified exceptions.

-- Reviewed in representation of the Hummingbot Foundation Technical DAO

**Review by aarmoa — CHANGES_REQUESTED** [review]

**Comment by lt7** [comment]
Closing, needs to be refactored to latest connector standards.

**Pull request closed without merging**